lib/tasks/manage.rake

```diff
@@ -1,6 +1,33 @@
 require "csv"
 
 namespace :manage do
+  desc "View all subscriptions for a subscriber"
+  task :view_subscriptions, [:email_address] => :environment do |_t, args|
+    email_address = args[:email_address]
+    subscriber = Subscriber.find_by_address(email_address)
+    abort("Cannot find any subscriber with email address #{email_address}.") if subscriber.nil?
+
+    results = subscriber.subscriptions.map do |subscription|
+      subscriber_list = SubscriberList.find(subscription.subscriber_list_id)
+      {
+        status: subscription.ended_at.present? ? "Inactive (#{subscription.ended_reason})" : "Active",
+        subscriber_list: "#{subscriber_list.slug} (id: #{subscriber_list.id})",
```

Comment:
Maybe also include the title of the subscriber list. This would be helpful as it would allow you to feed back to the user which lists they're subscribed to as each email they receive will have the title mentioned at the top of the email.

Reply:
I did originally do that but it made the table very wide. Happy to add it back in!